homeassistant/components/mazda/device_tracker.py

```diff
@@ -0,0 +1,58 @@
+"""Platform for Mazda device tracker integration."""
+from homeassistant.components.device_tracker import SOURCE_TYPE_GPS
+from homeassistant.components.device_tracker.config_entry import TrackerEntity
+
+from . import MazdaEntity
+from .const import DATA_COORDINATOR, DOMAIN
+
+
+async def async_setup_entry(hass, config_entry, async_add_entities):
+    """Set up the device tracker platform."""
+    coordinator = hass.data[DOMAIN][config_entry.entry_id][DATA_COORDINATOR]
+
+    entities = []
+
+    for index, _ in enumerate(coordinator.data):
+        entities.append(MazdaDeviceTracker(coordinator, index))
+
+    async_add_entities(entities)
+
+
+class MazdaDeviceTracker(MazdaEntity, TrackerEntity):
+    """Class for the device tracker."""
+
+    @property
+    def name(self):
+        """Return the name of the entity."""
+        vehicle_name = self.get_vehicle_name()
+        return f"{vehicle_name} Device Tracker"
+
+    @property
+    def unique_id(self):
+        """Return a unique identifier for this entity."""
+        return f"{self.vin}_device_tracker"
```

Comment:
We don't need to include the platform name since the entity registry is aware of both the integration domain and the platform domain.